sourmash_lib/signature_json.py

```diff
@@ -70,6 +70,8 @@ def _json_next_signature(iterable,
     ksize = d['ksize']
     mins = d['mins']
     n = d['num']
+    if n == 4294967295:               # load legacy signatures where n == -1
```

Comment:
personal preference, but this is a place where hex formatting makes it clearer (`0xffffffff` instead of `4294967295`)